db/stop_database.go

```diff
@@ -188,16 +182,11 @@ func doThreeStepPostgresExit(process *psutils.Process) error {
 		}
 		exitSuccessful = waitForProcessExit(process, 5*time.Second)
 	}
-	if !exitSuccessful {
-		// MURDER
-		err = killProcessTree(process)
-		if err != nil {
-			return err
-		}
-		exitSuccessful = waitForProcessExit(process, 5*time.Second)
-	}
 
 	if !exitSuccessful {
+		log.Printf("[ERROR] ** > Failed to stop service\n")
```

Comment:
do we want all this logging in production code?

Reply:
should it be at TRACE instead? so we can enable it as required?

Reply:
This code should never be executed . If it comes here, then we will get this as a BIG RED FLAG!!!

Reply:
all the same, I don't think we want a wall of *ERROR* lines - we could print a single error line, and then format the remaining information in to an error report which is printed underneath

Reply:
Updated